docs/src/main/asciidoc/logging.adoc

```diff
@@ -87,8 +87,8 @@ public class MyService {
 <2> Invoke the desired logging methods on the `log` object.
 
 
-[[logging-with-panache]]
-=== Logging with Panache
+[[simplified-logging]]
+=== Simplified Logging
```

Comment:
```suggestion
=== Simplified logging
```

We're supposed to use sentence-style capitalizaion for titles now (after I learned to love the Chicago Manual of Style title capitalization :joy:).

Reply:
Oh, well spotted. I'd followed the original casing, but in the original Panache was the product name, not a word, which is why it was capitalised.